src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -12,6 +12,15 @@ void npe(String a, String b) {
   a.length();
```

Comment:
Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
Severity: INFO
File: /home/travis/build/tomasbjerre/violations-test/src/main/java/se/bjurr/violations/lib/example/MyClass.java L19
 

Must have at least one statement.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>